src/boxed.tsx

```diff
@@ -17,6 +17,8 @@ type Props = {
     /** "data-" prefix is automatically added. For example, use "testid" instead of "data-testid" */
     dataAttributes?: DataAttributes;
     'aria-label'?: string;
+    width?: number | string;
+    height?: number | string;
```

Comment:
please, can you review MediaCard, DataCard and SnapCard to use these new props instead of a className to set the width and height?